pages/linux/ncdu.md

```diff
@@ -0,0 +1,19 @@
+# ncdu
+
+> Disk usage analyzer with an ncurses interface.
+
+- Analyze current working directory:
```

Comment:
Perhaps `Analyze the current working directory` would read better?

Reply:
done